### PR TITLE
small update to main readme

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tihomir@temporal.io>

### CODE REVIEW

**Review by robholland — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
-# background-checks
+# Background Checks Application
 
-background-checks is an example Temporal Application project.
+This project showcases the implementation of a background checks application with Temporal.
+The application is written in the [Go programming language](https://go.dev/), and uses the 
+Temporal [Go SDK](https://github.com/temporalio/sdk-go).
+See the list of available Temporal SDKs [here](https://docs.temporal.io/).
```
Comment (flossypurse) [inline README.md:6]:
```suggestion
This is an example [Temporal Application](https://docs.temporal.io/docs/content/what-is-a-temporal-application) project.

The application is written using the [Temporal Go SDK](https://github.com/temporalio/sdk-go).
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
-background-checks is an example Temporal Application project.
+This project showcases the implementation of a background checks application with Temporal.
+The application is written in the [Go programming language](https://go.dev/), and uses the 
+Temporal [Go SDK](https://github.com/temporalio/sdk-go).
+See the list of available Temporal SDKs [here](https://docs.temporal.io/). 
 
-This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
+The use case for this application is inspired by the blog post 
+["How Temporal simplified Checkr workflows"](https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows/)
```
Comment (flossypurse) [inline README.md:9]: I actually would prefer not to change this.

The project narrative goes into the inspiration - we want to call out what the use case is directly to align expectations up front.

Comment (flossypurse) [reply README.md:8]: I meant, we should keep this:
```
This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 
-background-checks is an example Temporal Application project.
+This is an example [Temporal Application](https://docs.temporal.io/docs/content/what-is-a-temporal-application) project.
 
-This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
+The application is written using the [Temporal Go SDK](https://github.com/temporalio/sdk-go).
 
-## How to
+The use case for this application is inspired by the blog post 
+["How Temporal simplified Checkr workflows"](https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows/)
```
Comment (flossypurse) [inline README.md:8]:
```suggestion
This project showcases how to implement a Temporal Application within the context of a [Long Running Human Driven Workflow](./docs/project-narrative.md/#what-is-a-long-running-human-driven-workflow) use case.
```

File: README.md [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+
+## How to use the application
 
 - [Guide to using this application](./docs/how-to-use.md)
 
-## Learning
+## Learn about application cores, requirements, and design/implementation decisions
```
Comment (flossypurse) [inline README.md:13]:
```suggestion
## Learn about the application
```

**Pull request merged**